packages/voter-dapp/src/ResolvedRequests.js

```diff
@@ -128,10 +129,10 @@ function ResolvedRequests({ votingAccount }) {
         return { filter: { roundId: showAllResolvedRequests ? undefined : indexRoundId }, fromBlock: 0 };
       }, [currentRoundId, showAllResolvedRequests])
     ) || [];
+
   useEffect(() => {
     setHasSpamRequests(false);
-
-    if (allResolvedEvents) {
+    if (allResolvedEvents && allResolvedEvents.length) {
```

Comment:
+1

Reply:
Can you not do `allResolvedEvents?.length`?

Reply:
interesting, yea i havent used this new syntax yet but its a good candidate for it. can update